webapp/locales/de.json

```diff
@@ -293,12 +293,14 @@
       },
       "columns": {
         "name": "Name",
-        "slug": "Alias"
+        "slug": "Alias",
+        "unblk": "Entblocken"
       },
       "empty": "Bislang hast du niemanden blockiert.",
       "how-to": "Du kannst andere Benutzer auf deren Profilseite über das Inhaltsmenü blockieren.",
       "block": "Nutzer blockieren",
-      "unblock": "Nutzer entblocken"
+      "unblock": "Nutzer entblocken",
+      "unblocked": " ist wieder entblockt"
```

Comment:
Couldn't we just always use the `unblock` translation and remove the `unblk` one?

Also, I am not sure I like the word `Entblocken`... Maybe `Freigeben`, `Entsperren` or `Zulassen` could be better options? @Tirokk what do you think? You're good with words. :smile: